pkg/controller/storagecluster/update.go

```diff
@@ -77,13 +79,31 @@ func (c *Controller) rollingUpdate(cluster *corev1.StorageCluster, hash string)
 		oldPodsToDelete = append(oldPodsToDelete, pod.Name)
 	}
 
+	// If px version is greater than 2.15 then delete px-api pods along with px pods
```

Comment:
I don't think we need changes from this file anymore. We had removed it in https://github.com/libopenstorage/operator/pull/1366.